src/action-creators/undoArchive.js

```diff
@@ -21,8 +18,8 @@ export const undoArchive = ({ originalPath, currPath, offset }) => dispatch => {
 
   const { getState } = store
 
-  const archiveContext = contextOf(pathToContext(currPath))
-  const contextOfArchive = splice(archiveContext, archiveContext.length - 1, 1)
+  const context = contextOf(pathToContext(currPath))
+  const archiveContext = contextOf(pathToContext(originalPath))
```

Comment:
You'll want to use `rootedContextOf` instead of `contextOf` here so that `archiveContext` gets set to `[ROOT]` instead of `[]`. `[]` is not a valid context.

You can test this out by archiving a thought in the root and then clicking "Undo".